resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnSchedulerBackend.scala

```diff
@@ -74,7 +74,7 @@ private[spark] abstract class YarnSchedulerBackend(
   private implicit val schedulerEndpointEC = ExecutionContext.fromExecutorService(
       ThreadUtils.newDaemonSingleThreadExecutor("yarn-scheduler-endpoint"))
 
-  /** Application ID. */
+  /** Application ID. This work for both cluster and client mode. */
```

Comment:
super nit: We dont really need to call this out

Reply:
Done